spec/paperclip/storage/s3_spec.rb

```diff
@@ -14,6 +14,21 @@ def aws2_add_region
     defined?(::Aws) ? { s3_region: 'us-east-1' } : {}
   end
 
+  context 'multithreaded initialization' do
+    before do
+    end
+    it "should not fail on missing constants" do
+      10.times.map do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.